src/code/z_parameter.c

```diff
@@ -163,50 +159,52 @@ static Gfx sScreenFillSetupDL[] = {
     gsSPEndDisplayList(),
 };
 
-s16 D_801BF9B0 = 0;
-f32 D_801BF9B4[] = {
+s16 sBButtonDoActionTextureScale = 0;
+f32 sBButtonDoActionTextureScales[] = {
+    // 100 is 1:1 scale, > 100 magnifies
     100.0f, // LANGUAGE_JPN
     109.0f, // LANGUAGE_ENG
     // Data missing for other languages?
 };
-s16 D_801BF9BC[] = {
-    0x226, // EQUIP_SLOT_B
-    0x2A8, // EQUIP_SLOT_C_LEFT
-    0x2A8, // EQUIP_SLOT_C_DOWN
-    0x2A8, // EQUIP_SLOT_C_RIGHT
+s16 sItemIconTextureScales[] = {
+    (s16)(1.074219 * (1 << 10)) >> 1, // EQUIP_SLOT_B
```

Comment:
```suggestion
    (s16)(1.074219f * (1 << 10)) >> 1, // EQUIP_SLOT_B
```
Could you add the `f`s to these numbers?